Applications/WebPortal/WebApp/library/source/pages/account_page/risk_page/risk_page.tsx

```diff
@@ -0,0 +1,164 @@
+import * as Nexus from 'nexus';
+import * as React from 'react';
+import { DisplaySize, HLine, PageWrapper } from '../../..';
+import { SubmissionInput } from '..';
+import { RiskParametersView } from '.';
```

Comment:
Going forward we will no longer import from ```'.'```. You can import from ```'../..'``` etc... but never import from ```'.'``` due to issues it causes with circular dependencies. So in this case you will need to explicitly import from ```'./risk_parameters_view'```.